README.md

```diff
@@ -31,29 +32,39 @@
     cp just-spaces/local_settings.example.py just-spaces/local_settings.py
     ```
 
-5. Create the database:
+5. Install NPM dependencies for the frontend:
```

Comment:
I'd love to have more robust documentation of what's going on here! `just-spaces` will be the first implementation of this approach, and I imagine the project repo will be a valuable model for others. 

I think you could include that documentation (e.g., links to useful resources, verbose explanations) in the step-by-step, or you could have a subsection that gives an overview.

Reply:
Hm, I thought I pushed this documentation up, but the last commit in this PR seems to have not included the changes. (Just changed editors, and I'm still getting used to its Git workflow.) I'll just push those docs straight to `master` if that's OK with you @reginafcompton, since it's just an edit to the README.

Reply:
Go right ahead! Thanks for the catch.